packages/services/f-feature-management/test/endToEnd.test.js

```diff
@@ -6,11 +6,13 @@ const { getBooleanValue, getIntegerValue, getStringValue } = featureManagement('
 const functionMap = { boolean: getBooleanValue, integer: getIntegerValue, string: getStringValue };
 
 describe.skip('End-to-End Tests', () => {
-    describe.each(expectations)('Expectation %# - ', ({ description, expectation, feature, valueType }) => {
+    describe.each(expectations)('Expectation %# - ', ({
+        description, expectation, feature, valueType
+    }) => {
         it(description, () => {
             const { value: expectedValue, experimentKey: expectedKey, experimentVariant: expectedVariant } = expectation;
-            const feature = functionMap[valueType](feature);
-            expect(feature.value).toBe(expectedValue);
+            const featureValue = functionMap[valueType](feature);
+            expect(featureValue.value).toBe(expectedValue);
```

Comment:
`featureValue.value` reads oddly to me. I'm assuming this is so you don't get a linting error for `feature` being redefined in a nested scope? I'd prefer reassigning `feature` to `featureName` above:
```
    describe.each(expectations)('Expectation %# - ', ({
        description, expectation, feature: featureName, valueType
    }) => {
```
and then continue to use `feature` within the test case.

Reply:
Yeah, you're right about why I renamed it, however it will soon also contain the experiment name and variant for tracking too.

However, I like your suggestion as `feature` is still a better name for the model, so will change.